src/ipyforcegraph/behaviors/scales.py

```diff
@@ -70,8 +89,10 @@ class Scheme(enum.Enum):
         allow_none=True,
     ).tag(sync=True)
 
-    domain: Tuple[float, float] = W.TypedTuple(
-        (0.0, 1.0),
+    domain: Tuple[float, float] = T.Tuple(
+        T.Float(),
```

Comment:
Added this more specific trait definitions, but happy to go back to `TypedTuple` if you think that's better.